exporter/collector/config.go

```diff
@@ -266,7 +266,7 @@ func generateClientOptions(ctx context.Context, clientCfg *ClientConfig, cfg *Co
 			// option.WithGRPCConn option takes precedent over all other supplied options so the
 			// following user agent will be used by both exporters if we reach this branch
 			dialOpts := []grpc.DialOption{
-				grpc.WithStatsHandler(&ocgrpc.ClientHandler{}),
+				// TODO: use meterProvider for gRPC instrumentation
```

Comment:
@aabmass @damemi The only thing left for this PR is to choose to use either the gRPC or go-contrib library here.  I tried to use the gRPC version, but it hasn't actually been released yet: https://pkg.go.dev/google.golang.org/grpc/stats/opentelemetry (notice the tag is just a commit).

@zasweq do you know if the grpc OpenTelemetry metrics plugin is ready to be used by other projects, or when you expect the first release to be tagged?